mm2src/mm2_bitcoin/script/src/sign.rs

```diff
@@ -437,7 +437,7 @@ impl TransactionInputSigner {
         input_index: usize,
         script_pubkey: &Script,
         sighashtype: u32,
-        _sighash: Sighash,
+        sighash: Sighash,
```

Comment:
Would appreciate if the comment block above this function could be updated to detail our implemention vs a standard implementation. The comment no longer makes sense as we are now making use of SIGHASH_SINGLE. 

It would also be pertinent to detail the potential dangers of SIGHASH_SINGLE. See https://hackmd.io/@matan/sighash_post

Reply:
c.c. @artemii235